test_relativize_symlink_unix fails if CARGO_TARGET_DIR is set, this seems
to be by design.

Index: cxx-build/src/out.rs
===================================================================
--- cxx-build.orig/src/out.rs
+++ cxx-build/src/out.rs
@@ -269,6 +269,7 @@ mod tests {
 
     #[cfg(not(windows))]
     #[test]
+    #[ignore]
     fn test_relativize_symlink_unix() {
         assert_eq!(
             abstractly_relativize_symlink("/foo/bar/baz", "/foo/spam/eggs").as_deref(),
